docs/api/select.md

```diff
@@ -157,6 +157,14 @@ Customizing the interface dialog should be done by following the Customization s
 
 However, the Select Option does set a class for easier styling and allows for the ability to pass a class to the overlay option, see the [Select Options documentation](select-option.md) for usage examples of customizing options.
 
+## Typeahead Component
+
+Typeahead or autocomplete functionality can be built using existing Ionic components. We recommend using an `ion-modal` to make the best use of the available screen space.
+
+import TypeaheadExample from '@site/static/usage/v7/select/typeahead/index.md';
+
+<TypeaheadExample />
```

Comment:
The Vue stackblitz doesn't show anything in the modal regardless of what's typed:
![vue](https://i.gyazo.com/781c013196569f8d7101e836d233a02c.png)

Happens in both v6 and v7.

Reply:
Fixed in https://github.com/ionic-team/ionic-docs/pull/2704/commits/ca83b10b20ac073f6055571d25082476e929f2fb